packages/x-date-pickers/src/locales/huHU.ts

```diff
@@ -0,0 +1,70 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+
+// maps TimeView to its translation
+const timeViews = {
+  hours: 'Óra',
+  minutes: 'Perc',
+  seconds: 'Másodperc',
+};
+
+const huHUPickers: Partial<PickersLocaleText<any>> = {
+  // Calendar navigation
+  previousMonth: 'Előző hónap',
+  nextMonth: 'Következő hónap',
+
+  // View navigation
+  openPreviousView: 'Előző nézet megnyitása',
+  openNextView: 'Következő nézet megnyitása',
+  calendarViewSwitchingButtonAriaLabel: (view) =>
+    view === 'year'
+      ? 'az évválasztó már nyitva, váltson a naptárnézetre'
+      : 'a naptárnézet már nyitva, váltson az évválasztóra',
+  inputModeToggleButtonAriaLabel: (isKeyboardInputOpen: boolean, viewType: 'calendar' | 'clock') =>
+    isKeyboardInputOpen
+      ? `szöveges beviteli nézet aktív, váltás ${viewType} nézetre`
```

Comment:
`viewType` will always be in english here and in turn will result in the following labels produced:
- calendar beviteli nézet aktív, váltás szöveges beviteli nézetre
- szöveges beviteli nézet aktív, váltás calendar nézetre
- clock beviteli nézet aktív, váltás szöveges beviteli nézetre
- szöveges beviteli nézet aktív, váltás clock nézetre

Adding a something along these lines would produce more correct result:

```ts
// maps PickersToolbar["viewType"] to its translation
const pickerViews = {
  calendar: 'naptár',
  clock: 'óra',
};
```

This would produce the following resulting translations instead:
- naptár beviteli nézet aktív, váltás szöveges beviteli nézetre
- szöveges beviteli nézet aktív, váltás naptár nézetre
- óra beviteli nézet aktív, váltás szöveges beviteli nézetre
- szöveges beviteli nézet aktív, váltás óra nézetre

Does it sound correct to you @noherczeg ?

Reply:
I've updated PR with the suggested solution.
If you find any issues, feel free to fire a new PR. 😉 
And again, thank you for the contribution and help @noherczeg! 🚀